src/utils/loadPath.ts

```diff
@@ -18,26 +22,28 @@ const loadArchives = async (authToken: string): Promise<FileEntry[]> => {
   });
 };
 
-// TODO - this is spoofed right now but will be replaced with appropriate SDK calls once
-// that part of the SDK is released.
-const loadArchive = async (authToken: string, archiveName: string): Promise<FileEntry[]> => [
-  generateFileEntry(
-    `${archiveName}/MyFiles`,
-    generateDefaultAttributes(fs.constants.S_IFDIR),
-  ),
-  generateFileEntry(
-    `${archiveName}/Shares`,
-    generateDefaultAttributes(fs.constants.S_IFDIR),
-  ),
-  generateFileEntry(
-    `${archiveName}/Public`,
-    generateDefaultAttributes(fs.constants.S_IFDIR),
-  ),
-  generateFileEntry(
-    `${archiveName}/Apps`,
+const getArchiveIdFromArchivePath = (archivePath: string): number => {
```

Comment:
In the spirit of #33 I should probably write some unit tests for this.

Reply:
A lot of this is moving to a new home in a future PR so I'm not going to test it here.